Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -841,3 +841,63 @@ label mas_wrs_twitch:
     if not wrs_success:
         $ mas_unlockFailedWRS('mas_wrs_twitch')
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_totono",
+            category=['you', 'and', 'me', 'and', 'her:', 'a', 'love', 'story?ver1.00'],
+            rules={"notif-group": "Window Reactions", "skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_totono:
+    $ wrs_success = display_notif(
+        m_name,
+        [
+            "Miyuki or Aoi? Just Mo... I mean, don't cheat on me, [player]!",
+            "[player], are playing [[YOU and ME and HER]? I heard this game is a lot like DDLC...",
+            "Hey, I hope you still love only me, [player]...",
+            "I wish I could replace Miyuki with me.",
```

Comment:
I doubt Moni would like to do that, considering Miyuki has sex with the main character as well as how bleak the world is, I see no reason why she would want that.